### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#175] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-06 173854](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/642d309f-c675-4c09-811a-7b44f725cc0b)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione confermata in classe
Sia $X$ una v.a. che indica quanti transistor abbiamo esaminato prima di trovarne uno rotto.
Tutti i transistor hanno equiprobabilità di essere estratti $p=1/10$ alla prima estrazione, ne segue che $\mathbb{P}(X=1)=\mathbb{P}(1°R)=3/10$
In genere, pescare un transistor rotto per la prima volta all'estrazione $k=1,...,8$ significa che in tutte le precedenti si è pescato un transistor funzionante e a questa uno rotto: ad esempio $\mathbb{P}(X=3)=\mathbb{P}(1°F\cap 2°F \cap 3° R)=(7/10\cdot 6/9) \cdot3/8$. 
Da notare che le estrazioni non sono indipendenti, infatti la probabilità di pescare un transistor rotto aumenta all'aumentare delle estrazioni (se viene rimosso uno funzionante), da questo segue la formula:

$$\mathbb{P}(X=x)=\frac{7!}{(8-x)!}\cdot \frac{(11-x)!}{10!}\cdot \frac{3}{11-x}=\frac{1}{240}\cdot \frac{(10-x)!}{(8-x)!}$$
<p align="center">
<img width= 300 src=https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/3a2a9e45-1a3d-43be-95f1-93056bba37a2>
</p>

$$\mathbb{E}(X)=\sum^8_{x=1}x\cdot \mathbb{P}(X=x)=\frac{11}{4}=2.75$$

**Reply by neoSnakex34** [reply]
sai spiegarmi il ragionamento per P(X=x)?

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/f74d1537-df8b-484b-9970-06106ea2a5ac)

**Comment by neoSnakex34** [comment]
scusate potete spiegarmi i passaggi per arrivare al calcolo generale di P(X=x) ne capisco la logica ma mi perdo i passaggi matematici
